tests/layer_tests/mo_python_api_tests/test_mo_convert_tf.py

```diff
@@ -688,7 +688,7 @@ def test_unnamed_saved_model_dir(self, ie_device, precision, ir_version, temp_di
 
     def test_zero_copy(self, ie_device, precision, ir_version, temp_dir):
         import tensorflow as tf
-        from openvino.tools.mo import convert_model
+        from openvino.tools.ovc import convert_model
```

Comment:
Do we fully switch to ovc in tests? Shouldn't we test both at least during transition period?

Reply:
OVC is the main tool that will be developed, MO will be frozen and deprecated. So it is preferable to test OVC when possible. 
Currently most tests in CI use tools.mo.convert_model. Only ovc unit tests and part of MO python API tests are switched to OVC.
Ideally we need to test both, but not sure that we should do it in CI and double CI time. It would be good to be able run tools.mo tests on demand when we change legacy functionality. Probably it is a task for validation team.